pkgs/development/python-modules/requests-unixsocket/default.nix

```diff
@@ -0,0 +1,24 @@
+{ lib, buildPythonPackage, fetchPypi, requests }:
+
+buildPythonPackage rec {
+  pname = "requests-unixsocket";
+  version = "0.1.5";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0k19knydh0fzd7w12lfy18arl1ndwa0zln33vsb37yv1iw9w06x9";
+  };
+
+  propagatedBuildInputs = [ requests ];
+
+  # has unpackaged test dependency pytest-capturelog, should be
```

Comment:
Would it be too big an effort to package pytest-capturelog?

Reply:
I think it would be unnecessary, since not only is that functionality now upstream, there's a more popular fork "pytest-catchlog" which is packaged. Also, there are other unpackaged test deps besides that one.